packages/react-native-bridge/ios/GutenbergWebFallback/FallbackJavascriptInjection.swift

```diff
@@ -68,11 +68,13 @@ public extension SourceFile {
     }
 }
 
-internal extension String {
+public extension String {
```

Comment:
We _may_ be able to revert this back to `internal`, because I had a thought about how we're calling the `jsScript() -> WKUserScript` function:

I noticed in the [WPiOS PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/17644/files#diff-4aadfc0278d3b8272613aa3de61efa1636c9d9bf943f5346e3ebb98b547098c2R112) that we used a string literal to construct the `window.injectCss` call and I'm thinking about potential pitfalls in the future if someone were to rename that injected function.

An idea I had was that we could take the [`SourceFile` extension](https://github.com/WordPress/gutenberg/pull/37221/files#diff-ec8720728c732299da7f328b24179bbbde3596fab310df48b6ffe64938b6356bL63) and move it to [`SourceFile.swift`](https://github.com/WordPress/gutenberg/blob/95929b58e4f3a6bdc3f2df3cb612858d5051952b/packages/react-native-bridge/ios/SourceFile.swift) where we could access its private variables and dynamically generate the necessary JavaScript depending on its type.

```
extension SourceFile {
    public func jsScript(with argument: String? = nil) throws -> WKUserScript {
        let content = try getContent()
        let formatted = String(format: content, argument ?? [])
        
        switch self.type {
        case .css:
            let injectCssScriptTemplate = "window.injectCss(`%@`)"
            return String(format: injectCssScriptTemplate, formatted).toJsScript()
        case .js, .json:
            return formatted.toJsScript()
        }
    }
}
```

This could keep the injection template string tightly coupled with the `inject-css` file, as well as simplify the `loadCustomStyles` call in the [WPiOS PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/17644/files#diff-4aadfc0278d3b8272613aa3de61efa1636c9d9bf943f5346e3ebb98b547098c2R111) to something like: `return try SourceFile(name: name, type: .css).jsScript(with: argument)`

I'm also OK with how it is now, so I wanted to get your thoughts. 🙇

Reply:
I really appreciate this insight into better programming practices, thank you 🙇‍♀️ What you've outlined all seems 
favourable to me and I've gone ahead to implement it on the Gutenberg side in https://github.com/WordPress/gutenberg/pull/37221/commits/0cda8b3e569faba16bc10aa39c60d44658e92268. (I'll implement the necessary changes on the iOS side when I make my way to that review!)